.github/workflows/rolling-release.yml

```diff
@@ -183,14 +183,14 @@ jobs:
         "C:\Program Files\7-Zip\7z.exe" a -r duckstation-windows-arm64-release.zip ./bin/ARM64/*
 
     - name: Upload arm64 release artifact
-      uses: actions/upload-artifact@v4
+      uses: actions/upload-artifact@v4.3.3
       with:
         name: "windows-arm64"
         path: "duckstation-windows-arm64-release*.zip"
 
 
   linux-build:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
Linux builders have to stay on 22.04, because otherwise 24.04 becomes the minimum for the AppImage.

Reply:
Got it, thanks for the heads up!
Reverted to ubuntu-22.04.